package.json

```diff
@@ -106,7 +106,7 @@
     "@openapitools/openapi-generator-cli": "1.0.13-4.3.1",
     "@types/file-saver": "2.0.1",
```

Comment:
Fix warnings of unused variables

Reply:
All the warnings are not related with files I changed